src/towers.effekt

```diff
@@ -0,0 +1,170 @@
+import mutable/array
+import src/benchmark
+import immutable/list
+
+//each list represents one smallerDisk.
+//TODO type alias, rename as "smallerDisk"
+//smallerDisk.next is the smallerDisk below => kellerstack
+type Disk = List[Int]
+
+
+// def printPiles(piles: Array[Disk]) = {
+//   println(pilesToString(piles))
+// }
+
+// def DiskToString(d: Disk): String = {
+//   "Disk("++helper(d)++")"
+// }
+
+// def helper(d: Disk): String = {
+//   d match {
+//     case Cons(size, next) => ""++show(size)++" " ++ helper(next)
+//     case Nil() => ""
+//   }
+// }
+
+// def assert(myBool: Boolean, s: String) = {
+//   if (not(myBool)) {
+//     panic("assertion failed:"++s)
+//   }
+// }
+
+// def pilesToString(piles: Array[Disk]): String = {
+//   var s = "["
+//   each(0, piles.size()){ i =>
+//     s = s ++ "\n\t" ++ DiskToString(piles.unsafeGet(i))
+//   }
+//   s = s ++ "]"
+//   s
+// }
+
+def Towers() = {
+
+  var piles: Array[Disk] = emptyArray();
+  var movesDone: Int = 0;
+
+  def pushDisk(piles: Array[Disk], smallerDisk: Disk, pileIdx: Int): Disk = {
```

Comment:
`smallerDisk` should be an `Int`. You could introduce an alias `type Disk = Int`.